### DISCUSSION TITLE
remote_state s3_bucket lifecycle rules

### DISCUSSION THREAD

**Discussion by georg-ikegps** [gruntwork-io/knowledge-base#324] [Help]
I am wondering how to add a lifecycle rule to my s3 remote state?

The bucket is versioned, and Security Hub keeps nagging me that I should add a lifecycle rule.
Is there a way to add the rule via terragrunt during setup?

Thank you for you help!

**Comment by denis256** [comment]
Hi,
afaik this is not supported now
